tardis/io/tests/test_config_reader.py

```diff
@@ -190,35 +196,55 @@ def test_plasma_nlte_section_config(tardis_config_verysimple_nlte):
 
     Parameter
     ---------
-        `tardis_config_verysimple` : YAML File
+        `tardis_config_verysimple_nlte` : YAML File
+        `nlte_raw_model` : A simple model
+        `nlte_atom_data` : An example atomic dataset
 
     Result
     ------
         Assertion based on validation for specified values
     """
-    conf = Configuration.from_config_dict(
-        tardis_config_verysimple_nlte, validate=True, config_dirname="test"
-    )
     tardis_config_verysimple_nlte["plasma"]["continuum_interaction"][
         "species"
     ] = [
         "He I",
     ]
     tardis_config_verysimple_nlte["plasma"]["nlte_ionization_species"] = ["H I"]
-    with pytest.raises(ValueError) as ve:
-        nlte_ionization_species = tardis_config_verysimple_nlte["plasma"][
-            "nlte_ionization_species"
-        ]
-
-        for species in nlte_ionization_species:
-            if not (
-                species
-                in tardis_config_verysimple_nlte["plasma"][
-                    "continuum_interaction"
-                ]["species"]
-            ):
-                raise ValueError("Nlte ionization species not in continuum.")
-    assert ve.type is ValueError
+    with pytest.raises(PlasmaConfigError) as ve:
+        plasma = assemble_plasma(
+            tardis_config_verysimple_nlte, nlte_raw_model, nlte_atom_data
+        )
```

Comment:
The new tests make much more sense than the old ones but aren't they strictly speaking plasma tests?
What do other people think?

Reply:
Nobody seems to care. Let's move on.